fix(legacy): use swc to transform esm to es5 (#20)
hCaptcha/hcaptcha-loader: README.md

| `sentry`          | Boolean     | No       | `-`                                | See enterprise docs.                                                                                                                                      |
| `custom`          | Boolean     | No       | `-`                                | See enterprise docs.                                                                                                                                      |

## Legacy Support
In order to support older browsers, a separate bundle is generated in which all ES6 code is compiled down to ES5 along with an optional polyfill bundle.

- `polyfills.js`: Provides polyfills for features not supported in older browsers.
- `index.es5.js`: **@hcaptcha/loader** package compiled for ES5 environments.

### Import Bundle(s)
Both bundles generated use IIFE format rather than a more modern import syntax such as `require` or `esm`.

```js
// Optional polyfill import
import '@hCaptcha/loader/dist/polyfills.js';
// ES5 version of hCaptcha Loader
import '@hCaptcha/loader/dist/index.es5.js';

> window['hCaptchaLoader']().then(function(hcaptcha) {

Reviewer: Couldn't we keep this as
**hCaptchaLoader.then** without the window[]?

Author: This can be used as
- `hCaptchaLoader()`
- `window.hCaptchaLoader()`
- `window['hCaptchaLoader']`

Third option works with TS out of the box, i'll update the readme

| `reportapi`       | String      | No       | `-`             | See enterprise docs.                                                                                                                                      |
| `sentry`          | Boolean     | No       | `-`             | See enterprise docs.                                                                                                                                      |
| `custom`          | Boolean     | No       | `-`             | See enterprise docs.                                                                                                                                      |

## Legacy Support

> In order to support older browsers, a seperate bundle is generated in which all ES6 code is compiled down to ES5 along with an optional polyfill bundle to extend functionality that is used common to modern browsers.

Reviewer 2: ```suggestion
In order to support older browsers, a separate bundle is generated in which all ES6 code is compiled down to ES5 along with an optional polyfill bundle.
```

- `polyfills.js`: Provides polyfills for features not supported in older browsers.
- `index.es5.js`: **@hcaptcha/loader** package compiled for ES5 environments.

### Import Bundle(s)

> Both bundles generated utilize IIFE format instead of modern importation syntax such as `require` or `esm`.

Reviewer 2: ```suggestion
Both bundles generated use IIFE format rather than a more modern import syntax such as `require` or `esm`.
```

```js
// Optional polyfill import
import '@hCaptcha/loader/dist/polyfills.js';
// ES5 version of hCaptcha Loader
import '@hCaptcha/loader/dist/index.es5.js';

hCaptchaLoader.then(function() {
    var element = document.createElement('div');
    // hCaptcha API is ready
    hcaptcha.render(element, {
        sitekey: 'YOUR_SITE_KEY',
        // Additional options here
    });
});

```

### CDN

> The hCaptcha Loader targetted for older browsers can also be imported via a CDN by leveraging UNPKG](https://www.unpkg.com/), see example below.

Reviewer 2: ```suggestion
The hCaptcha Loader targeted for older browsers can also be imported via CDN by using UNPKG](https://www.unpkg.com/), see example below.
```
